Apps/phgitlab/gitlab.json

```diff
@@ -0,0 +1,450 @@
+                    "data_type": "string",
+                    "column_name": "status",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.description",
+                    "data_type": "string",
+                    "example_values": [
+                        "Sample trigger description"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.token",
+                    "data_type": "string",
+                    "column_order": 3,
+                    "example_values": [
+                        "123456abcdef"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.id",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "run pipeline",
+            "identifier": "run_pipeline",
+            "description": "Execute a pipeline on a particular branch of a project",
+            "verbose": "Execute a pipeline on a particular branch of a project. GitLab CI/CD must be configured on the project for this action to succeed.",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "project_id": {
+                    "description": "The ID of the GitLab project",
+                    "data_type": "numeric",
+                    "required": true,
+                    "primary": false,
+                    "contains": [
+                      "gitlab project id"
+                    ],
+                    "order": 0
+                },
+                "trigger_token": {
+                    "description": "The trigger token required for executing a pipeline",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": false,
+                    "contains": [
+                      "gitlab trigger"
+                    ],
+                    "order": 1
+                },
+                "branch": {
+                    "description": "The branch containing the .gitlab-ci.yml file to run. Defaults to 'master'",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": false,
```

Comment:
Primary should be true in this case because the contains list is not empty

Reply:
I set the value to `true` in [this commit](https://github.com/peterbertel/phantom-apps/commit/d480700f682bec546797af4a5bb5d72534d59575).